app/controllers/events_controller.rb

```diff
@@ -1,6 +1,7 @@
 class EventsController < ApplicationController
 
   def index
+    @events = Event.all.paginate(page: params[:page])
   end
```

Comment:
Okay, that's one thing that I wanted to bring up tomorrow.  Right now the events index is the sell page and that's why it's all of the events.